<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="sim" default="true" />
  <arg name="perception" default="false" />
  <arg name="world_file" value="$(find world_model_data)/data/worlds/default.xml" />
  <arg name="world_channel" value="world_model" />
  <arg name="fact_channel" value="add_facts" />
  <arg name="seed" default="0" />
  
  <!-- Set robot-specific paramters -->
  <param name="world_frame" type="str" value="world" />
  <param name="torso_frame" type="str" value="base" />
  <param name="arm_base_frame" type="str" value="base_link" />
  <param name="ee_frame" type="str" value="robotiq_85_base_link" />
  <param name="tool_offset_frame" type="str" value="[0.15, 0.003,0,0,-1.5707,0]" />
  <param name="controller_ns" type="str" value="trajectory_controller" />

  <param name="gripper_open_pos" type="double" value="0.0" />
  <param name="gripper_close_pos" type="double" value="0.8" />
  
  <!-- <node name="action_client" -->
  <!--       pkg="complex_action_client" -->
  <!--       type="arm_client_ur5_robotiq_2F_85.py" -->
  <!--       args="-\-sim" -->
  <!--       output="screen" -->
  <!--       required="true" -->
  <!--       /> -->

  <!-- <node name="behavior_tree" -->
  <!--       pkg="behavior_tree" -->
  <!--       type="dynamic_behavior_tree.py" -->
  <!--       output="screen" -->
  <!--       required="true" /> -->

  <!-- Params -->
  <param name="world_channel" value="$(arg world_channel)" />
  <param name="fact_channel" value="$(arg fact_channel)" />
  <param name="obj_channel" value="add_objects" />
  
  <node name="world_model"
        pkg="manip_e"
        type="world_model.py"
        args="--seed $(arg seed)"
        output="screen"
        required="true" />
    
</launch>
  
  
  
